src/main/resources/sla/sla-did-sales-conditions-template.json

```diff
@@ -0,0 +1,137 @@
+[
+  {
+    "name": "lockPayment",
+    "timelock": 0,
+    "timeout": 0,
+    "contractName": "LockPaymentCondition",
+    "functionName": "fulfill",
+    "parameters": [
+      {
+        "name": "_did",
+        "type": "bytes32",
+        "value": "{parameter.assetId}"
+      },
+      {
+        "name": "_rewardAddress",
+        "type": "address",
+        "value": "{contract.EscrowPaymentCondition.address}"
+      },
+      {
+        "name": "_tokenAddress",
+        "type": "address",
+        "value": "{parameter.tokenAddress}"
+      },
+      {
+        "name": "_amounts",
+        "type": "uint256[]",
+        "value": {parameter.amounts}
+      },
+      {
+        "name": "_receivers",
+        "type": "address[]",
+        "value": {parameter.receivers}
+      }
+    ],
+    "events": [{
+      "name": "Fulfilled",
+      "actorType": "publisher",
+      "handler": {
+        "moduleName": "lockPaymentConditon",
+        "functionName": "fulfillAccessCondition",
+        "version": "0.1"
+      }
+    }]
+  },
+
+  {
+    "name": "transferDID",
+    "timelock": 0,
+    "timeout": 0,
+    "contractName": "TransferDIDCondition",
+    "functionName": "fulfill",
+    "parameters": [{
+      "name": "_documentId",
+      "type": "bytes32",
+      "value": "{parameter.assetId}"
+    },
+    {
+      "name": "_receiver",
+      "type": "address",
+      "value": ""
+    }
+    ],
+    "events": [{
+      "name": "Fulfilled",
+      "actorType": "publisher",
+      "handler": {
+        "moduleName": "transferDID",
+        "functionName": "fulfillEscrowPaymentCondition",
+        "version": "0.1"
+      }
+    },
+      {
+        "name": "TimedOut",
+        "actorType": "consumer",
+        "handler": {
+          "moduleName": "access",
+          "functionName": "fulfillEscrowPaymentCondition",
+          "version": "0.1"
+        }
+      }
+    ]
+  },
+
+  {
+    "name": "escrowPayment",
+    "timelock": 0,
+    "timeout": 0,
+    "contractName": "EscrowPaymentCondition",
+    "functionName": "fulfill",
+    "parameters": [
+      {
+        "name": "_did",
+        "type": "bytes32",
+        "value": "{parameter.assetId}"
+      },
+      {
+        "name": "_amounts",
+        "type": "uint256[]",
+        "value": {parameter.amounts}
+      },
+      {
+        "name": "_receivers",
+        "type": "address[]",
+        "value": {parameter.receivers}
```

Comment:
should the value be in quotes?

Reply:
this file is just a template, the output generated include in this case an array of strings